__tests__/endpoints.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+
+
+
+beforeEach(() => {
+    return seed(data);
+  });
+  
+  afterAll(() => {
+    return db.end();
+  });
+  
+  
+  describe("api/topics", () => {
+      test("status check responds 200", () => {
+          return request(app)
+          .get("/api/topics")
+          .expect(200)
+    });
+      test("status check responds 200", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.topics).toEqual([
```

Comment:
This test is not very solid as if we change the test data it will fail. It might be better to loop through topics and check for each property. Remember to also check the length if you follow this approach 🙂